emission/core/wrapper/suggestion_sys.py

```diff
@@ -499,25 +527,25 @@ def calculate_yelp_server_suggestion_nominatim(uuid):
                     #Not sure to include the amount of carbon saved
                     #Still looking to see what to return with this message, because currently my latitude and longitudes are stacked together in one string
                     # insert_into_db(tripDict, i, yelp_suggestion_trips, uuid)
-                    return {'message' : message, 'method': 'bike', 'rating': str(ratings_bus[a]), 'businessid': a}
+                    return {'message' : message, 'method': 'bike', 'rating': str(ratings_bus[a]), 'businessid': a, "tripid": str(cleaned_trips.iloc[i]["_id"])}
```

Comment:
I think you can just return the `tripid` the serializer should handle it

Reply:
I changed it to string, because I believe Sam said the phone uses strings as tripids?

Reply:
@valin1 that's what I meant by serializer. The output JSON serializer that converts JSON -> string converts any tripid in the JSON -> string